terraform/constants.go

```diff
@@ -0,0 +1,7 @@
+package terraform
+
+const (
+	TF           = ".tf"
+	TF_VARS      = ".tfvars"
+	TF_VARS_JSON = ".tfvars.json"
```

Comment:
```suggestion
	TF           = ".tf"
```
nit: we are not really using these yet, do we?

Reply:
we don't, those are leftovers from when we looked at extracting vars from `.tfvars` files.